### PR TITLE
[ETFE-3056]: added MovementReviewAnswers page/controller/navgiation

### PR DESCRIPTION
**Pull request by LadonJackson**

### CODE REVIEW

**Review by LewisMitchell — DISMISSED** [review]
The movement section on the task list doesn't route to this new review answers page, it probably needs to?

File: test/views/sections/movement/MovementReviewAnswersViewSpec.scala [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+      implicit val doc: Document = Jsoup.parse(view(
+        form = form,
+        list = SummaryListViewModel(Seq.empty),
+        onSubmitCall = testOnwardRoute
+      ).toString())
+
+      behave like pageWithExpectedElementsAndMessages(Seq(
+        Selectors.title -> English.title,
+        Selectors.h1 -> English.heading,
```
[the remark this answers is not in the document]
Comment (LewisMitchell) [reply test/views/sections/movement/MovementReviewAnswersViewSpec.scala:54]: Never mind, I didn't see the selector below this line

**Comment by LewisMitchell** [comment]
This now includes content changes to the invoice details page and a movement details check answers page.

**Review by MJCallahanPage — APPROVED** [review]
LGTM 🟢 - I've ran it locally and manually exploratory tested, works as I would expect now.

**Pull request merged**